README.md

```diff
@@ -219,6 +235,42 @@ Em seguida, acesse o link para confirmar as autorizações
   response.url
 ```
 
+### Estorno de Transações
+
+Você pode estornar pagamentos que as transações estiverem com status: Paga (3), Disponível (4), Em disputa (5).
+
+```ruby
+  refund = PagSeguro::Refund.new
+  refund.transaction_code = "D5D5BE444148407891E497B421975599"
+
+  response = refund.register
+
+  if response.errors.any?
+    puts response.errors.join("\n")
+  else
+    puts "=> REFUND RESPONSE"
+    puts response.result
+  end
+```
+
+### Cancelamento de Transações
+
+Você pode cancelar transações que estiverem com status: Aguardando pagamento ou Em análise.
+
+```ruby
+  cancellation = PagSeguro::TransactionCancellation.new
+  cancellation.transaction_code = "AFB8FCF29496401681257C1ECE3A98FF"
+
+  response = cancellation.register
```

Comment:
This example broken sample because the method register will return a *boolean* value and bellow you'll try run the errors and result method from this value (bool). So I recommend you to update the method register in the `PagSeguro::TransactionCancellation` class, something like this:
```ruby
...

def register
  request = Request.post("transactions/cancels", api_version, params)
  Response.new(request, self).serialize
end

...
```
This way you'll you keep the standard used in other classes like `lib/pagseguro/refund.rb`